CHANGES.rst

```diff
@@ -18,6 +18,8 @@
 
 - Fix wiping out substep parameters settings when using Step.call [#28]
 
+- Add keyword 'logcfg' to Step.call() to set logging configuration [#32]
```

Comment:
Stpipe 3.0 was released already. This needs to be moved to a new section.

Reply:
Sorry about that. Should I add a new section header? Will this cause merge rebasing for future stpipe PRs?